utils/tests/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+find_package(cmocka)
+
+add_executable(test_utils
+        test_comparison.c
+        "$<TARGET_OBJECTS:utils>"
+)
+target_include_directories(test_utils PRIVATE
+        ${CMOCKA_INCLUDE_DIR}
+        ../include
+)
+target_link_libraries(test_utils
+        m
```

Comment:
Can you explain what `m` is? Is it a cmocka thing, something else I've forgotten about, or a typo?

Reply:
Ahhh right this is `-lm` written in CMake! Got it.